crates/eigen-cli/src/lib.rs

```diff
@@ -2,15 +2,36 @@
     html_logo_url = "https://github.com/Layr-Labs/eigensdk-rs/assets/91280922/bd13caec-3c00-4afc-839a-b83d2890beb5",
     issue_tracker_base_url = "https://github.com/Layr-Labs/eigensdk-rs/issues/"
 )]
+use crate::eigen_address::ContractAddresses;
 use alloy_contract::Error as ContractError;
 use alloy_json_rpc::RpcError;
 use alloy_transport::TransportErrorKind;
+use args::{Commands, EigenKeyCommand};
+use ark_serialize::SerializationError;
+use convert::store;
+use eigen_crypto_bls::error::BlsError;
+use generate::KeyGenerator;
+use operator_id::derive_operator_id;
 use thiserror::Error;
 pub mod args;
```

Comment:
separate the module declarations form imports

Reply:
Done!